routing/world_graph.hpp

```diff
@@ -41,10 +41,9 @@ class WorldGraph final
              std::shared_ptr<EdgeEstimator> estimator);
 
   void GetEdgeList(Segment const & segment, bool isOutgoing, bool isLeap,
```

Comment:
Please, add a comment to |isEnding| parameter. Does it mean ending start or end segment?

Reply:
Done. More comments in world_graph.cpp.